Build cuda without cuda?

Hello,

I have build my project, and resolve its dependencies when building a [Singularity](https://www.sylabs.io/docs/) image (think Docker without sudo rights for executing).
I need sudo rights to build the image, so I do it on my machine, without GPU, and run the image on a cluster with GPU (there is a special flag to expose the GPU driver).

When I build my image, I take the PyTorch version with GPU, even though I don't have any when I do the install. This does not give me any problem and I can run GPU code when I deploy.

Now, resolving `pytorch_scatter` (_et. al._), [this](https://github.com/rusty1s/pytorch_scatter/blob/cfa8554a3ebac581dea1df7fb1d6064bcca8c3cd/setup.py#L12) resolves to False and the Cuda extension is not built. I get at run-time:
```
/usr/lib/python3.7/site-packages/torch_scatter/utils/ext.py:5: in <module>
    import scatter_cuda
E   ModuleNotFoundError: No module named 'scatter_cuda'
```

Is it possible to change the test `torch.cuda.is_available()` in [setup.py](https://github.com/rusty1s/pytorch_scatter/blob/cfa8554a3ebac581dea1df7fb1d6064bcca8c3cd/setup.py#L12) to something less strong, _i.e._ ensuring that the extension can be built but not that there is a full Cuda run-time?
I'm not even sure that's possible...

Thanks

--- a/setup.py
+++ b/setup.py
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 import torch
-from torch.utils.cpp_extension import CppExtension, CUDAExtension
+from torch.utils.cpp_extension import CppExtension, CUDAExtension, CUDA_HOME
 
 ext_modules = [
     CppExtension(
@@ -9,7 +9,7 @@ ext_modules = [
 ]
 cmdclass = {'build_ext': torch.utils.cpp_extension.BuildExtension}
 
-if torch.cuda.is_available():
+if CUDA_HOME is not None:
     ext_modules += [
         CUDAExtension('scatter_cuda',
                       ['cuda/scatter.cpp', 'cuda/scatter_kernel.cu'])

